app/src/shared/lib/hooks/useToggled.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback, useState } from 'react';
+
+export const useToggled = (initialState?: boolean) => {
```

Comment:
```suggestion
export const useToggle = (initialState?: boolean) => {
```
или
```suggestion
export const useIsToggled = (initialState?: boolean) => {
```
function = verb